data/data.json

```diff
@@ -8753,6 +8925,10 @@
    "minRKEVersion": "1.2.14-rc0",
    "minRancherVersion": "2.5.11-rc0"
   },
```

Comment:
Shouldn't `v1.18.20-rancher1-3` be added in here too?

Reply:
@PennyScissors addressed in https://github.com/rancher/kontainer-driver-metadata/pull/751, this PR will be auto-updated on merging that. Can you review?